plugins/ruby-foreman-templates/debian/control

```diff
@@ -8,7 +8,7 @@ Homepage: https://github.com/theforeman/foreman_templates
 
 Package: ruby-foreman-templates
 Architecture: all
-Depends: ${misc:Depends}, bundler, foreman (>= 1.12.0), git
+Depends: ${misc:Depends}, bundler, foreman (>= 1.14.1), git
```

Comment:
it would work with 1.14 but it would be missing "model" in metadata, but I guess that's minor thing (literally :-) since 1.14.1 is out anyway